src/Runner.Common/Logging.cs

```diff
@@ -32,15 +32,30 @@ public class PagingLogger : RunnerService, IPagingLogger
         private string _pagesFolder;
         private IJobServerQueue _jobServerQueue;
 
+        // For Results
+        public static string BlocksFolder = "blocks";
+
+        // 2 MB
+        public const int BlockSize = 2 * 1024 * 1024;
```

Comment:
move these public variables up to where other public variables live.

Reply:
any reason not keep using 8MB?

Reply:
Yeah Azure Append Blob supports 4MB blocks only - and since this is really minimum 8MB, we need to reduce the size to be safe.